Laravel-whib/tests/Unit/MapPinTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Models\MapPin;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class MapPinTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function it_can_create_a_map_pin()
+    {
+        // Arrange
+        $data = [
+            'pin_name' => 'Test Pin',
+            'description' => 'Test description',
+            'favourite' => false,
+            'latitude' => 123.456789,
+            'longitude' => -45.678901,
+            'user_id' => "4325",
+            'category' => "hotel"
+        ];
+
+        // Act
+        $mapPin = MapPin::create($data);
```

Comment:
Still, this is more of a 'idk' type of test. Unit test shouldn't use database, even im memory. If you wanna use memory db, create feature tests which test whole routes. 
That's why unit tests require these SOLID rules, to make it testable etc. If you wanna make good unit test, which prof. Rewak won't have heart attack when he sees it, read about mock and try to make few separete files out of your controller (like service, repository or something).